app/presenters/publishing_api_presenters/contact.rb

```diff
@@ -0,0 +1,121 @@
+require_relative "../publishing_api_presenters"
+
+class PublishingApiPresenters::Contact
```

Comment:
I've amended [this commit](https://github.com/alphagov/whitehall/pull/2632/commits/1f884ba2139aec91ad87bc3eed664f359204ad13) to explain why this change doesn't follow the `ItemPresenter` inheritance pattern TL;DR, because contacts do not have a `base_path`.